### PR TITLE
Feature/sqlite

### PR DESCRIPTION
**Pull request by tommy2d**
- Enabled SQLite by removing dependencies on rowCount() calls.
- Added explicit exception in case FIDO2Scope turns out to be NULL.
- Actually initialized credentialIdEncoded array.

### CODE REVIEW

File: www/regprocess.php [hunk 1]
```diff
@@ -31,6 +31,10 @@
     throw new Exception("Attempt to register new token in unacceptable context.");
 }
 
+$fido2Scope = ( $state['FIDO2Scope'] === null ? $state['FIDO2DerivedScope'] : $state['FIDO2Scope'] );
```
Comment (tvdijen) [inline www/regprocess.php]: The extra spaces will break PSR-12 compliancy

**Comment by tvdijen** [comment]
Looking good!

**Comment by tommy2d** [comment]
Is there anything I need to do now in order to get this merged?

**Comment by tvdijen** [comment]
Nope, I just wanted to give others a chance to review

**Comment by restena-sw** [comment]
Since I don't use SQLite to try it out myself, I'm happy for @tvdijen to merge this.

**Comment by tvdijen** [comment]
I took care of it already

**Comment by tommy2d** [comment]
Great! I’m currently in Myanmar for a few weeks, without a device with a hardware keyboard, so rebasing would have been a pain.

**Pull request merged**